src/services/notification/index.js

```diff
@@ -176,13 +181,14 @@ class NotificationService {
 		const userId = (params.account || {}).userId || params.payload.userId;
 
 		const options = {
-			uri: `${serviceUrls.notification}/notifications/`
-                + `?user=${userId}&${toQueryString(params.query)}`,
+			uri:
+				`${serviceUrls.notification}/notifications/` +
```

Comment:
Codacy found an issue: ['+' should be placed at the beginning of the line.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)